src/esn.calendar.libs/app/components/event/form/event-form.directive.js

```diff
@@ -1,3 +1,5 @@
+require('./event-form.controller');
+
 'use strict';
```

Comment:
https://www.w3schools.com/js/js_strict.asp

"Strict mode is declared by adding "use strict"; to the beginning of a script or a function."

Should we move it to the top? Or wait until polish?

Reply:
We have this syntax in many places also. Need to discuss with others to have the decision, then create a ticket. Not sure we should do it in the polish.